src/components/Table.tsx

```diff
@@ -213,6 +215,16 @@ export default class Table<T> extends React.Component<
 		return some(this.state.checkedItems, { [rowKey]: identifier });
 	}
 
+	isHighlighted(item: T) {
+		const rowKey = this.props.rowKey;
+		if (!rowKey) {
+			return false;
+		}
+
+		const identifier = item[rowKey];
+		return this.props.highlightedRow === (identifier as any);
```

Comment:
How about changing this to allow highlighting more than one rows? Sounds symmetric to how how you can have more than one checked rows.

Reply:
Good idea